File: python/nt_solutions/denoisingadv_9_sure/exo6.py
plt.figure(figsize = (5,5))
tmp = np.min(E)
i = np.argmin(E)
lambd = lambda_list[i]
hf = h(f,lambd)
imageplot(clamp(hf), "SNR = %.1f dB" %snr(f0, hf))

File: julia/NtSolutions/denoisingadv_9_sure/exo6.jl
figure(figsize = (5, 5))
tmp = minimum(E)
val, i = findmin(E)
lambd = lambda_list[i]
hf = h(f, lambd)
imageplot(clamP(hf), @printf("SNR = %.1f dB", snr(f0, hf)))
